MANUAL.md

```diff
@@ -29,6 +29,36 @@ Cloud Foundry.
     assign a pushed application that did not specify its memory
     requirements, explicitly.  Defaults to `256`.
 
+  - `default_app_disk_in_mb` - How much disk (in megabytes) to
+    assign a pushed application that did not specify its memory
+    requirements, explicitly. Defaults to `1000`.
+
+  - `uaa_lockout_failure_count` - Amount of failed UAA login attempts
+    before lockout.
+
+  - `uaa_lockout_failure_time_between_failures` - How much time
+    (in seconds) in which `uaa_lockout_failure_count` must occur in 
+    order for account to be locked. Defaults to `1200`.
+  
+  - `uaa_lockout_punishment_time` - How long (in seconds) the account
+    is locked out for violating `uaa_lock_failure_count` within 
+    `uaa_lockout_failure_time_between_failures`. Defaults to `300`.
+
+  - `uaa_refresh_token_validity` - How long (in seconds) a CF refresh
+    is valid for. Defaults to `2592000`.
+
+  - `cf_branding_product_logo` - A base64 encoded image to display on
+    the web UI login prompt. Defaults to `nil`.
+
+  - `cf_branding_square_logo` - A base64 encoded image to display
+    in areas where a smaller logo is necessary. Defaults to `nil`.
+
+  - `cf_footer_legal_text` - A string to display in the footer,
+    typically used for compliance text. Defaults to `nil`.
+
+  - `cf_footer_links` - A YAML list of links to enumerate in the footer
+    of the web UI. Defaults to `nil`
```

Comment:
How are the links specified?  As raw HTML?  As an object containing link text and URL?  How would an operator use this?